libs/core/src/lib/toolbar/toolbar.component.ts

```diff
@@ -0,0 +1,203 @@
+import {
+    Component,
+    ViewEncapsulation,
+    AfterViewInit,
+    AfterViewChecked,
+    ChangeDetectionStrategy,
+    ElementRef,
+    ViewChild,
+    OnInit,
+    forwardRef,
+    ContentChildren,
+    QueryList,
+    HostListener,
+    ChangeDetectorRef,
+    Renderer2,
+    Input
+} from '@angular/core';
+import { ToolbarItemDirective } from './public_api';
+import { applyCssClass, CssClassBuilder } from '../utils/public_api';
+
+const ELEMENT_MARGIN = 8;
```

Comment:
We should use getComputedStyle() to get these so we don't have to rely on constants that could potentially change in the future

https://stackoverflow.com/a/6338234

Reply:
The problem with getComputedStyle() is the performance, I tried that method and it's not fast